.github/workflows/publish.yml

```diff
@@ -0,0 +1,19 @@
+name: Publish
+
+on: [push, pull_request]
```

Comment:
How about using `create` or `release` here?

https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#create
https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#release

Reply:
I don't have much time available. By all means, please edit this pr to your liking

Reply:
I do not have permission to access the settings page of this repo or set npm token in the github secrets. I will temporarily remove the workflow file, and then manually publish the new version of the landscape theme on npm.